src/app/components/about/about.module.js

```diff
@@ -1,9 +1,12 @@
 import aboutRun from './about.run';
 import aboutConfig from './about.config';
+import aboutControllers from './about.controllers';
 
 const aboutModule = angular
-  .module('ionicSeed.about', [])
-  .run(aboutRun)
-  .config(aboutConfig)
+    .module('ionicSeed.about', [
+        'ionicSeed.about.controllers'
```

Comment:
This is importing the entire controllers module. Perhaps the actual controller should be something more specific than AboutController.